src/main/scala/com/advancedtelematic/campaigner/db/Campaigns.scala

```diff
@@ -336,10 +336,8 @@ protected [db] class CampaignStatusTransition(implicit db: Database, ec: Executi
     for {
       affected <- devicesWithStatus(DeviceStatus.values - DeviceStatus.requested - DeviceStatus.rejected)
       finished <- devicesWithStatus(Set(DeviceStatus.successful, DeviceStatus.failed, DeviceStatus.cancelled))
-      cancelled <- devicesWithStatus(Set(DeviceStatus.cancelled))
       requested <- devicesWithStatus(Set(DeviceStatus.requested))
-      total <- devicesWithStatus(DeviceStatus.values)
-      wasCampaignCancelled = currentCampaignStatus.exists(_ == CampaignStatus.cancelled)
+      wasCampaignCancelled = currentCampaignStatus.contains(CampaignStatus.cancelled)
```

Comment:
The whole method is kind of weird. From what i see it should be `isFinished(...): Boolean`

Reply:
Indeed. Refactored this in a new PR at #138 since this doesn't seem so minor anymore.

Reply:
should we close this one and move to the new one?

Reply:
Yes, just that this one is rebased on #138 now. So only the second commit of this branch should be reviewed here.